llvm/docs/CommandGuide/lit.rst

```diff
@@ -535,6 +535,7 @@ TestRunner.py:
  %{fs-tmp-root}          root component of file system paths pointing to the test's temporary directory
  %{fs-sep}               file system path separator
  %t                      temporary file name unique to the test
+ %basename_s             The last path component of %s
```

Comment:
I see this is consistent with the already existing basename_t, but I feel that it would make a lot more sense if this was consistent with `%{s:real}` and `%{/s:regex_replacement}` as below, i.e. `%{s:basename}`.

Reply:
Let me look into changing this. I agree this makes sense.

Reply:
I created: https://github.com/llvm/llvm-project/pull/111062, will start review once checks pass.